### PR TITLE
[BEAM-1077] @ValidatesRunner Test in Python Postcommit

### PR DESCRIPTION
**Pull request by markflyhigh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

Add ValidatesRunner tests to postcommit. Currently running tests on dataflow service.

Testing subset of ValidatesRunner tests on dataflow service is done. Here is the [Job1 link](https://pantheon.corp.google.com/dataflow/job/2016-12-02_11_53_43-6218835747597492543?project=google.com:clouddfe) and [Job2 link](https://pantheon.corp.google.com/dataflow/job/2016-12-02_11_50_13-12814563807408026450?project=google.com:clouddfe).

### CODE REVIEW

**Comment by markflyhigh** [comment]
+R: @aaltay

**Comment by aaltay** [comment]
Have you successfully run the script with changes? This script only runs in post commit tests.

**Comment by markflyhigh** [comment]
@aaltay fixed some errors in script and successfully running test by running run_postcommit.sh

**Comment by aaltay** [comment]
@markflyhigh Mark, is this ready to review? Have you ran the updated script successfully?

**Comment by markflyhigh** [comment]
@aaltay yes, it's ready for review. I used another project to runt the script since I can't use apache-beam-testing directly.

**Comment by markflyhigh** [comment]
+CC: @jasonkuster

File: sdks/python/run_postcommit.sh [hunk 1]
```diff
@@ -54,29 +54,43 @@ echo ">>> RUNNING DIRECT RUNNER py-wordcount"
...
 PROJECT=apache-beam-testing
 
 # Create a tarball
 python setup.py sdist
 
+SDK_LOCATION=$(find dist/apache-beam-sdk-*.tar.gz)
+
+# Run ValidatesRunner tests on Google Cloud Dataflow service
+python setup.py nosetests \
+  -a ValidatesRunner --processes=4 --process-timeout=360 \
```
Comment (aaltay) [inline sdks/python/run_postcommit.sh]: What is the meaning of `--processes=4 --process-timeout=360`

File: sdks/python/run_postcommit.sh [hunk 2]
```diff
@@ -54,29 +54,43 @@ echo ">>> RUNNING DIRECT RUNNER py-wordcount"
...
+  --test-pipeline-options=" \
+    --runner=BlockingDataflowPipelineRunner \
+    --project=$PROJECT \
+    --staging_location=$GCS_LOCATION/staging-validatesrunner-test \
+    --sdk_location=$SDK_LOCATION \
+    --job_name=$JOBNAME_VR_TEST\
+    --num_workers=1"
```
Comment (aaltay) [inline sdks/python/run_postcommit.sh]: Make sure the job output appears in the logs, it makes debugging easier (see `cat job_output` below`)

Comment (markflyhigh) [inline sdks/python/run_postcommit.sh]: done

File: sdks/python/run_postcommit.sh (hunk 1 above)
Comment (markflyhigh) [inline sdks/python/run_postcommit.sh]: comment added.

**Comment by markflyhigh** [comment]
PTAL @aaltay

**Comment by aaltay** [comment]
LGTM, thank you.

**Comment by markflyhigh** [comment]
@robertwb

**Pull request closed without merging**